usercommandmgmt/configurable.py

```diff
@@ -0,0 +1,12 @@
+#contains all the configurable fields for this cog
+
+#set to false to disable the pre-command creation moderator approval system
+command_moderation = False
+
+#specifies the number of commands granted to specific roles
+role_cmd_limits = {
+    "Tier 1 Simp" : 1,
+    "Tier 2 Simp": 3,
+    "Phillip Paik himself": 5
+}
+
```

Comment:
This works, but have you thought about using Config to store this? This way you:
1. Don't need to have a dedicated file.
2. Allows for configuration on a per-guild basis (as opposed to globally).
3. It can be changed on the fly if you have the commands to manipulate them.